background/get-userscripts.js

```diff
@@ -210,6 +211,20 @@ chrome.alarms.onAlarm.addListener((alarm) => {
   }
 });
 
+chrome.webRequest.onResponseStarted.addListener(
+  (request) => {
+    const identity = createCsIdentity({ tabId: request.tabId, frameId: request.frameId, url: request.url });
```

Comment:
wouldn't it be easier to do
```suggestion
    const identity = createCsIdentity(request);
```

Reply:
Related: https://jakearchibald.com/2021/function-callback-risks/